src/tests/test_utils/common_test_utils/include/common_test_utils/ov_plugin_cache.hpp

```diff
@@ -39,9 +39,10 @@ class PluginCache {
     PluginCache();
     ~PluginCache() = default;
 
-    std::mutex g_mtx;
     bool disable_plugin_cache;
     std::shared_ptr<ov::Core> ov_core;
+
```

Comment:
potentially, we can lock `get()` function as a provider singleton instance

Discussed offline